rust/azure_iot_operations_services/src/azure_device_registry.rs

```diff
@@ -411,29 +406,24 @@ impl From<adr_name_gen::DeviceEndpointSchema> for DeviceEndpoints {
                 .collect(),
             None => HashMap::new(),
         };
-        let outbound_assigned;
-        let outbound_unassigned;
+        let mut outbound_assigned = HashMap::new();
+        let mut outbound_unassigned_map = HashMap::new();
```

Comment:
```suggestion
        let mut outbound_unassigned = HashMap::new();
```
needed to include map? Otherwise it'll be easier to build the struct at the end